corehq/apps/export/const.py

```diff
@@ -142,3 +142,6 @@ class SharingOption(object):
         EXPORT_ONLY,
         EDIT_AND_EXPORT,
     )
+
+
+UNKNOWN_EXPORT_OWNER = 'unknown'
```

Comment:
Should this be translated since it is being shown to users?

Reply:
why not just `ugettext_lazy` the constant, then you can remove that `if` statement?

Reply:
There's a formatting different (see the ```div```) between user names and "Unknown"

Reply:
@proteusvacuum I let a lot of time go by before addressing this - let me know if you want me to ping my new code buddy for review.